pcapgo/capture.go

```diff
@@ -0,0 +1,48 @@
+package pcapgo
```

Comment:
please copy over copyright notices to top of new files.

Reply:
Done